nvflare/app_common/np/np_model_persistor.py

```diff
@@ -26,14 +25,11 @@
 
 
 def _get_run_dir(fl_ctx: FLContext):
-    engine = fl_ctx.get_engine()
-    if engine is None:
-        raise RuntimeError("engine is missing in fl_ctx.")
-    job_id = fl_ctx.get_prop(FLContextKey.CURRENT_RUN)
+    workspace = fl_ctx.get_workspace()
+    job_id = fl_ctx.get_job_id()
```

Comment:
Should we change get_job_id to use CURRENT_JOB_ID, instead the old RUN_NUM?

Reply:
This is from old days, and some examples already look for RUN_NUM directly. So we can't change the key now.